### PR TITLE
Call the lambdas!

### PR DESCRIPTION
**Pull request by samfrench**

### CODE REVIEW

File: test/ingress/web_test.exs [hunk 1]
```diff
@@ -3,7 +3,8 @@ defmodule Ingress.WebTest do
   use Plug.Test
 
   alias Ingress.Web
-  #@opts Ingress.init([])
+
+  import Mock
```
Comment (ettomatic) [inline test/ingress/web_test.exs]: Not 100% convinced about this. 
The Elixir community is quite strong on this and I'm wondering if we'll end up using libraries not widely used. Could make sense to investigate a bit different approaches.

One interesting reading from the creator of Elixir:
http://blog.plataformatec.com.br/2015/10/mocks-and-explicit-contracts/

File: config/config.exs [hunk 2]
```diff
@@ -32,4 +32,9 @@ config :ingress, http_port: 7080
 config :ingress, env:       Mix.env
 config :ingress, origin:    System.get_env("INGRESS_ORIGIN")
 
+config :ingress, lambda_presentation_role:  System.get_env("LAMBDA_PRESENTATION_ROLE")
```
Comment (james-bowers) [inline config/config.exs]: perhaps this should go in `prod.exs`?

File: test/ingress/web_test.exs (hunk 1 above)
Comment (james-bowers) [reply test/ingress/web_test.exs]: That blog post links to https://github.com/plataformatec/mox - so I'll look into that

Comment (ettomatic) [reply test/ingress/web_test.exs]: discussed face 2 face, let's merge like it's now and well spend some time to agree on a different/better strategy

**Pull request merged**